### ISSUE TITLE
5.3.1 has breaking change

### ISSUE THREAD

**Issue by tylerlong** [mobxjs/mobx-react#574]
```
index.min.js:1 Uncaught ReferenceError: process is not defined
    at index.min.js:1
    at i (index.min.js:1)
    at index.min.js:1
    at index.min.js:1
    at index.min.js:1
```

I am using CDN version: https://unpkg.com/mobx-react@5.3.1/index.min.js

5.2.8 version is fine.

**Comment by xaviergonz** [comment]
that's weird, the actual piece of code that fails comes from react itself... it seems like react is not being applied properly as an external by rollup

**Comment by xaviergonz** [comment]
ok, false alarm (kind of), it comes from react-is, which is a dependency of hoist-non-react-statics

@mweststrate should mobx react do something like 
`
if (typeof "process" === "undefined") {
  process = { env: {} }
}
`

on index.js before all imports?

**Comment by mweststrate** [comment]
Released as 5.3.2
